### ISSUE TITLE
Ability to switch between diffrent MPRIS Instances

### ISSUE THREAD

**Issue by solotitan** [Jas-SinghFSU/HyprPanel#99]
**Is your feature request related to a problem? Please describe.**

If more than one MPRIS instance is registered, like firefox steaming a podcast for example, and I have spotifyd running in the background.. you can't switch from one to the other as far as I know.

**Describe the solution you'd like**

Being able to toggle between or pick different sources would be useful.  

**Additional context**

mpris-proxy gives

```
player org.mpris.MediaPlayer2.firefox.instance_1_24 at :1.24 found
player org.mpris.MediaPlayer2.spotifyd.instance828 at :1.0 found
```

**Comment by Shubham-550** [comment]
Does the media slider work for Firefox on your system?

**Comment by solotitan** [comment]
No, it only has the ability to play and pause whatever is on firefox, if that's that what you mean.

**Comment by Jas-SinghFSU** [comment]
Implemented in #533
